db/migrate/20210823214444_create_tastings.rb

```diff
@@ -0,0 +1,14 @@
+class CreateTastings < ActiveRecord::Migration[6.0]
+  def change
+    create_table :tastings do |t|
+      t.string :title
+      t.string :description
```

Comment:
We should make the description a text area instead.

Reply:
Good Call. Just made the requested change